types/react-native-channel-plugin/index.d.ts

```diff
@@ -0,0 +1,577 @@
+// Type definitions for react-native-channel-plugin 0.7
+// Project: https://www.channel.io
+// Definitions by: Han Lee <https://github.com/hahnlee>
+//               : Yeonggyun Kim <https://github.com/CXZ7720>
```

Comment:
you would have to have this person explicit ACK to be listed here as the owner, as automation process sends notifications to package owners.
you could also just give a person explicit credits, instead of referencing in header

Reply:
Oh, I didn't know comments has such a function. First of all, it seems that I am the only one in this header.